app/pages/project/home.jsx

```diff
@@ -84,21 +76,14 @@ export default class ProjectHomePage extends React.Component {
 
   renderResearcherWords() {
     const avatarSrc = this.state.researcherAvatar || '/assets/simple-avatar.png';
-    let quote;
-
-    if (this.props.project.researcher_quote) {
-      quote = <span>&quot;{this.props.project.researcher_quote}&quot;</span>;
-    } else {
-      quote = <Translate component="span" content="researchQuote.default" />;
-    }
 
     return (
       <div className="project-home-page__researcher-words">
         <h4>Words from the researcher</h4>
```

Comment:
Yep, but this doesn't need to be addressed until we've added the ability to switch languages for a project.